exercises/rectangles/src/example/java/RectangleCounter.java

```diff
@@ -15,13 +15,13 @@ int countRectangles(final String[] rawGrid) {
 
     private static final class Grid {
 
-        private enum Entry {
+        private enum Element {
             CORNER,
             HLINE,
             VLINE,
             SPACE;
 
-            private static Entry fromChar(final char rawGridEntry) {
+            private static Element fromChar(final char rawGridEntry) {
```

Comment:
> I'm sick, so apologies for the abbreviated reply ...

Unacceptable; sickness is for the weak. 😷 

Oooohhh I love `Tile` too!  I dig the physicality of it.  Seems like the best names are those that are the most concrete but not overly-so.

Hope you're feeling better, soon, Stuart! 🤒